sys/src/9/boot/boot.c

```diff
@@ -298,14 +300,16 @@ static void catstuff(void)
 	ifd = open("#t/eia0", ORDWR);
 	if (ifd < 0) {
 		print("can't open eia0!!!\n");
-		return;
-	}
-	print("open: ifd %d\n", ifd);
-	while (1) {
-		sleep(100000);
-	}
+	} else {
```

Comment:
why not replace the return with an 
exits("can not open eia0");

that would be a one line change

Very nice catch by the way :-)

Reply:
could do, though we should also get rid of the print at the end of the function, since it'll never be reached.

Reply:
done